quickstarts/microsoft.logic/logic-app-create/metadata.json

```diff
@@ -6,18 +6,36 @@
   "summary": "Create a Consumption logic app with an ARM template.",
   "githubUsername": "jeffhollan",
   "docOwner": "ecfan",
-  "dateUpdated": "2022-04-27",
+  "dateUpdated": "2022-07-28",
   "type": "QuickStart",  
   "tags": {
     "subscriptiontype": "Azure4Student",
+    "id": 21,
+    "rank": 7,
     "resource": "Logic Apps, MySQL",
     "os": "Linux, Logic App",
```

Comment:
Incorrect name, should be "Azure Logic Apps" or just "Azure" as the field is "os".